app/postgis.py

```diff
@@ -0,0 +1,218 @@
+import binascii
```

Comment:
ST_AsBinary works, and I think the combination of ST_AsBinary and ST_AddBand,ST_CreateEmptyRaster would have allowed me to store offdb rasters. But it would have been a "hackhish" way to do it, plus I'm not sure it would have played 100% nice with Django's ORM. 

Here the from_pgraster and to_pgraster methods go from HEXWKB --> raster info (including the path of the offdb raster) and raster info --> HEXWKB. Once I have the path of the offdb raster I can initialize the GDALRaster object, which is the standard object in GeoDjango to reference raster fields.

I think I will ping the Django maintainers and see if they are interested in incorporating some of these changes into GeoDjango core.